src/com_tjvendors/admin_language/en-GB/en-GB.com_tjvendors.ini

```diff
@@ -245,3 +245,29 @@ COM_TJVENDORS_LOAD_BOOTSTRAP_DESC="If your site template is standard bootstrap t
 
 ; Since v1.3.5
 COM_TJVENDORS_VENDORFEE_INVALID_CURRENCY="Please enter Currency"
+
+; Since v1.4.0
+COM_TJVENDORS_ADDRESS="Details"
+COM_TJVENDORS_FORM_LBL_ADDRESS="Address"
+COM_TJVENDORS_FORM_DESC_ADDRESS="Enter address"
+COM_TJVENDORS_FORM_LBL_COUNTRY="Country"
+COM_TJVENDORS_FORM_DESC_COUNTRY="Select country"
+COM_TJVENDORS_FORM_LBL_REGION="Region"
+COM_TJVENDORS_FORM_DESC_REGION="Select region"
+COM_TJVENDORS_FORM_LBL_CITY="City"
+COM_TJVENDORS_FORM_DESC_CITY="Select city"
+COM_TJVENDORS_FORM_LBL_OTHER_CITY="Other City"
+COM_TJVENDORS_FORM_DESC_OTHER_CITY="Enter other city"
```

Comment:
@deepgawade89 Add a description as. If you don't find your city in the list add the name here